src/App.tsx

```diff
@@ -2,25 +2,39 @@ import React, { ChangeEventHandler, useState } from 'react';
 import './App.css';
 
 function App() {
-const[player,setPlayer]=useState<number>(0);
-
-const changePlayer = (player:number) =>{
-  if(player === 0)
-  setPlayer(1);  
+  interface player{
+    playerOne:number,
+    playerTwo:number,
+    playerUnasigned:number
+  }
+  const[player,setPlayer]=useState<number>(0);
```

Comment:
Player type is a number and this although works is not nice. Replace it with an enum. By doing so on lines 16-19 you will have something like:
const changePlayer=(player: Player)=>{
  if (player === Player.FIRST)
  setPlayer(Player.SECOND);
  console.log("apasat");
}

First/Second ar just some suggestions. You can use One/Two or even Foo/Bar